### PR TITLE
Use canvas to do scanning; remove need for web worker.

### PR DESCRIPTION
**Pull request by dlongley**

### CODE REVIEW

File: Spinner.vue [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<template>
+  <div class="spinner" />
+</template>
+<script>
+/*!
+  * Copyright (c) 2021 Digital Bazaar, Inc. All rights reserved.
+  */
+'use strict';
```
Comment (mattcollier) [inline Spinner.vue:8]:
```suggestion
```

File: test/config.js [hunk 2]
```diff
@@ -15,15 +15,11 @@ config.views.bundle.packages.push({
   manifest: path.join(__dirname, 'package.json')
 });
 
-const WorkerPlugin = require('worker-plugin');
-
+// FIXME: can we remove this entirely now?
```
Comment (mattcollier) [inline test/config.js:18]: I believe so, yes.  If webpack can build the browser bundle without this, we're good.

**Review by mattcollier — APPROVED** [review]
TY!

File: test/config.js (hunk 2 above)
Comment (dlongley) [reply test/config.js]: Removed.

File: Spinner.vue (hunk 1 above)
Comment (dlongley) [reply Spinner.vue]: Removed.

**Comment by dlongley** [comment]
@mattcollier, @JoshDunnDev -- I'm turning this back over to you guys to take from here.

**Pull request merged**